contrib/templates/experimental/scala-job/template/databricks.yml.tmpl

```diff
@@ -0,0 +1,52 @@
+# This is a Databricks asset bundle definition for {{.project_name}}.
+# See https://docs.databricks.com/dev-tools/bundles/index.html for documentation.
+bundle:
+  name: {{.project_name}}
+
+workspace:
+  host: {{.custom_workspace_host}}
+  artifact_path: {{.artifacts_dest_path}}/${workspace.current_user.short_name}
```

Comment:
It wasn't really clear from the template that this would be appended (but it makes sense!). Could you clarify that in the template?

And it would be good to include `${bundle.name}/${bundle.target}` here. The `${workspace.current_user.short_name}` is also relevant for user-level isolation, though only for the `dev` target. `prod` is a singleton (which is enforced by `mode: production`).